spec/connect/cli_spec.rb

```diff
@@ -293,9 +293,11 @@
       subject.new(argv)
     end
 
-    it 'output help on help flag' do
+    it 'outputs help on help flag with no line longer than 80 characters' do
       argv = %w{--help}
-      expect_any_instance_of(subject).to receive(:puts)
+      expect_any_instance_of(subject).to receive(:puts) do |option_parser|
+        expect(option_parser.instance_variable_get(:@opts).to_s.split("\n").map(&:length).max).to be <= 80
```

Comment:
Well, the instance of OptionParser has a `@opts` variable whose `to_s` method builds the help output. not the most obvious thing, I agree :wink:
But do you really insist to have a custom spec helper just for this one occurrence, where the current implementation already does what it should? 
And where your proposed solution does not? :stuck_out_tongue: I just tried it, and it just doesn't capture the output and I don't wanna spend time debugging some random code from StackOverflow.